File: Leetcode/1822. Sign of the Product of an Array/solution.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        count_negative_numbers = 0

        for num in nums:
            if num < 0:
                count_negative_numbers += 1
                continue
            if num == 0:
                return 0
        
        return 1 if count_negative_numbers % 2 == 0 else -1

File: Leetcode/1822. Sign of the Product of an Array/solution.rs
impl Solution {
    pub fn array_sign(nums: Vec<i32>) -> i32 {
        let mut count_negative_numbers = 0;

        for num in nums {
            if num < 0 {
                count_negative_numbers += 1;
                continue;
            }
            if num == 0{
                return 0
            }
        }

        return if count_negative_numbers % 2 == 0 {1} else {-1}
    }
}